frontend/packages/ux-editor/src/components/config/RuleComponent.tsx

```diff
@@ -146,70 +148,62 @@ class Rule extends React.Component<IRuleComponentProps, any> {
           </div>
           {this.state.ruleConnection.selectedFunction ? (
             <>
-              <div className='form-group a-form-group mt-2'>
-                <h2 className='a-h4'>
+              <div>
+                <h2 className={classes.subTitle}>
                   {this.props.t('ux_editor.modal_configure_rules_configure_input_header')}
                 </h2>
                 {Object.keys(this.props.ruleModelElements[selectedMethodNr].inputs).map(
-                  (key: any, index: any) => {
+                  (key: any) => {
                     const paramName = key;
                     return (
```

Comment:
If you came over a code that is using any type, we should refactor the type to be something strict. Within this case we can simplify the code like the suggestion below, hence we replaced the any with string and avoided one variable assignment.
```suggestion
                {Object.keys(this.props.ruleModelElements[selectedMethodNr].inputs).map((paramName: string) => (
```

Reply:
I did not get exactly what do you mean by this suggestion.

Reply:
{Object.keys(this.props.ruleModelElements[selectedMethodNr].inputs).map(
                  (paramName: string) => { **<-- this changed name to paramName with type of string**
                    const paramName = key; **<-- This can be removed**

Reply:
ok I fixed it.